dev-docs/makefile.md

```diff
@@ -8,16 +8,27 @@ We are unable to use the actual `Makefile` name in this repo; because we build t
 
 ## Usage
 
+For convenience it helps to make a symlink from `tasks.mak` to `Makefile`. There is a target to do this, to run it:
+```
+make -f tasks.mak makefile_symlink
+```
+
+Now you can call the targets simply with the `make` command without needing to specify the `tasks.mak` file. The file `Makefile` is gitignored, since we don't want to push it to the repo for the reasons stated in the previous section. 
+
 To run something from the Makefile, you will need to give `make` the `-f` (file) flag with a path to `tasks.mak`. For example, to run the `test` target:
```

Comment:
I think this can be deleted or moved up to be the first line of the Usage section.

Reply:
I wanted to leave this here since the symlink should be the first thing a user should do.